sirepo/package_data/template/controls/parameters.py.jinja

```diff
@@ -115,6 +116,19 @@ def _optimize_polyfit(variable_count):
 {% endif %}
 
 {% if controlSettings_operationMode == 'madx' %}
+def check_twiss_success():
+    if not {{ isTrackingSim }}:
+        return True
```

Comment:
Rather than have the generated code look like this:
```python
def check_twiss_success():
    if not False:
        return True
    ...
```
I'd change the generated code so it never calls check_twiss_success() unless isTrackingSim is true - move the python "if" to a jinja "if".

Reply:
changed it